src/integration-test/system-integration-tests.ts

```diff
@@ -135,24 +159,20 @@ describe('system-integration', () => {
 
       describe('configuring notifications', () => {
 
-        async function testNotificationConfiguration(
+        function testNotificationConfiguration(
           configuration: NotificationConfiguration,
-          projectId: null | number = null,
-          teamId: null | number = null,
+          responseJson: any,
```

Comment:
So far I haven't typed any of the JSON API format responses. I feel that if we want to do that it should be done consistently across the integration tests and planned a bit beforehand, since those typings already exist in minard-ui.